hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -0,0 +1,155 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "entropy_src",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  interrupt_list: [
+    { name: "es_noise_valid"
+      desc: "Asserted when noise source bits are available."}
+    { name: "es_noise_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ]
+  regwidth: "32",
+  registers: [
+    { name: "ESCONF",
+      desc: "ENTROPY_SRC configuration register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "Master enable",
+          desc: "Enable ENTROPY_SRC noise generation function."
+        }
+      ]
+    },
+    { name: "ESREV",
+      desc: "ENTROPY_SRC revision register",
+      swaccess: "ro",
+      hwaccess: "hro",
+      fields: [
+        { bits: "23:16",
+          name: "ENTROPY_SRC Chip Type",
+          desc: "Read of this register shows the type of chip using this ENTROPY_SRC HW.",
+          resval: "0x1"
+        }
+        { bits: "15:8",
+          name: "ENTROPY_SRC HW revision",
+          desc: "Read of this register shows the level of the ENTROPY_SRC HW."
+          resval: "0x2"
+       }
+        { bits: "7:0",
+          name: "ENTROPY_SRC ABI revision",
+          desc: "Read of this register shows the level of the ENTROPY_SRC ABI."
+          resval: "0x1"
+        }
+      ]
+    },
+    { name: "ESMINENT",
+      desc: "ENTROPY_SRC min-entropy estimate register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "31:0",
+          name: "ENTROPY_SRC Min-entropy",
+          desc: '''Read of this register provides the expected value of assessed min-etropy, 
+                 expressed as a single precision floating point number. This value is 1-x, 
+                 where x is the entropy per bit. For example, a full entropy source, as defined 
+                 by NIST 800-90-B, would have an x value of less than 2**(-64). This number 
+                 should be given as a lower bound for the entropy generated under normal operation.
+                 ''',
+          resval: "0x123ab"
+        }
+      ]
+    },
+    { name: "ESNOISE",
+      desc: "ENTROPY_SRC noise source",
+      swaccess: "ro",
+      hwaccess: "hrw",
+      hwext: "true",
+      hwre: "true",
+      fields: [
+        { bits: "31:0" 
+          desc: "A read of this register provides noise needed for an entropy source."
+        }
+      ]
+    },
+    { name: "ESCTRL",
+      desc: "ENTROPY_SRC control register",
+      swaccess: "rw",
```

Comment:
Might want this as `wo` so that SW doesn't have to clear it again after writing?

Reply:
Not sure about intent here. In general, all control and configuration registers should be readable for software debug. This is general standard practice.

Reply:
OK